lib/rucio/client/downloadclient.py

```diff
@@ -233,7 +236,10 @@ def download_pfns(self, items, num_threads=2, trace_custom_fields={}, traces_cop
             item['scope'] = did_scope
             item['name'] = did_name
             item['sources'] = [{'pfn': pfn, 'rse': rse}]
-            dest_file_path = os.path.join(dest_dir_path, did_name)
+            did_path_name = did_name
+            if self.extract_scope_convention == 'belleii' and did_name.startswith('/'):
```

Comment:
extract_scope_convention can be None (as per config_get above), then this will equality check will fail

Reply:
@bari12 maybe we should think of a way of abstracting this? having vo specific checks inside the generic download() is maybe not so good